ts/editor/rich-text-input/RichTextInput.svelte

```diff
@@ -200,45 +201,48 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     setupLifecycleHooks(api);
 </script>
 
-<div class="rich-text-input" on:focusin={setFocus} {hidden}>
-    {#if $content.length === 0}
-        <div
-            class="rich-text-placeholder"
-            style:font-family={$fontFamily}
-            style:font-size={$fontSize + "px"}
-            style:direction={$direction}
-        >
-            {$description}
-        </div>
-    {/if}
-
-    <RichTextStyles
-        color={$pageTheme.isDark ? "white" : "black"}
-        fontFamily={$fontFamily}
-        fontSize={$fontSize}
-        direction={$direction}
-        callback={stylesResolve}
-        let:attachToShadow={attachStyles}
-        let:stylesDidLoad
-    >
-        <div
-            bind:this={richTextDiv}
-            class={className}
-            class:night-mode={$pageTheme.isDark}
-            use:attachShadow
-            use:attachStyles
-            use:attachContentEditable={{ stylesDidLoad }}
-            on:focusin
-            on:focusout
-        />
-
-        {#await Promise.all([richTextPromise, stylesDidLoad]) then _}
-            <div class="rich-text-widgets">
-                <slot />
+<Collapsible collapsed={hidden}>
```

Comment:
I wonder if instead of using Collapsible inside RichTextInput (and PlainTextInput), you could also apply it in the user component, and pass in `hidden` with `bind:`. It's something I'd like to try out.

Reply:
That was easier than expected :smile: https://github.com/ankitects/anki/pull/2002/commits/9988c000c16f5ec3d7fbbc84abc1265f35aead9d